src/App.js

```diff
@@ -1127,9 +1119,10 @@ export class App extends React.Component<AppProps, AppState> {
                                 </a>
                             </h1>
                             <IconButton
+                                className="App__header-keyboardMenuIcon"
                                 ariaLabel={this.props.intl.formatMessage({ id: 'KeyboardInputModal.ShowHide.AriaLabel' })}
                                 onClick={this.handleKeyboardModalToggle}
-                                onKeyDown={this.handleKeyboardMenuIconKeydown}
+                                onKeyDown={this.handleKeyboardModalToggle}
```

Comment:
For handleKeybaordModalToggle event handler, we can do the same as handleKeyDownWorldIcon, where it just sets showKeyboardModal state to true.

Reply:
I see what you mean, the modal itself handles setting it the other way.  Done.